index.yaml

```diff
@@ -3014,21 +2990,7 @@ entries:
       catalog.cattle.io/namespace: cattle-logging-system
       catalog.cattle.io/release-name: rancher-logging-crd
     apiVersion: v1
-    created: "2021-09-29T18:13:52.239402-04:00"
-    description: Installs the CRDs for rancher-logging.
-    digest: a82697b4a9517f25d27ab54551ecd2915075bfd7867fdcf01a7f0039e316cac2
-    name: rancher-logging-crd
-    type: application
-    urls:
-    - assets/rancher-logging/rancher-logging-crd-100.0.1+up3.12.0.tgz
-    version: 100.0.1+up3.12.0
-  - annotations:
-      catalog.cattle.io/certified: rancher
-      catalog.cattle.io/hidden: "true"
-      catalog.cattle.io/namespace: cattle-logging-system
-      catalog.cattle.io/release-name: rancher-logging-crd
-    apiVersion: v1
-    created: "2021-08-25T15:50:58.609939054-07:00"
+    created: "2021-09-30T14:07:17.987241483-04:00"
```

Comment:
I believe are missing the package now with this merge commit.
We should probably perform an interactive rebase (`git rebase`) and then manually fix the confict.
cc @paynejacob

Reply:
The change I'm pulling in from dev-v2.6 regenerates the index.yaml. Should I not accept the incoming change? That's what I did using the git UI

Reply:
Ok done